README.md

```diff
@@ -46,16 +46,28 @@ The following Braintree libraries offer official Swift Package Manager support:
 * `BraintreeCard`
 * `BraintreeCore`
 * `BraintreeDataCollector`
+    * `KountDataCollector`
 * `BraintreePaymentFlow`
 * `BraintreePayPal`
-* `BraintreeThreeDSecure` **
+* `BraintreeThreeDSecure`
 * `BraintreeUnionPay`
 * `BraintreeVenmo`
 * `PayPalDataCollector`
+    * `PPRiskMagnes`
 
-** - To use the the `BraintreeThreeDSecure` library via SPM, you must [manually include](#manually-including-a-framework) the `CardinalMobile.framework` located in the `Frameworks` dir.
+#### BraintreeDataCollector
 
-#### Manually including a framework
+To use the `BraintreeDataCollector` library, you must also include the `KountDataCollector` library via SPM.
+
+#### PayPalDataCollector
+
+The `BraintreePaymentFlow`, `BraintreeThreeDSecure`, `BraintreePayPal`, and `BraintreeVenmo` libraries all require both the `PayPalDataCollector` and `PPRiskMagnes` libraries be included via SPM.
```

Comment:
This is making me wonder if we need a full breakdown of all the frameworks that are required for each payment method / flow. We'll definitely have that in the dev doc guides for each payment method, once those are updated for v5. I'm not sure if we also need it in the README?

Reply:
This is a good question 🤔 . It definitely is a bit confusing.

Would we want to include instructions per package manager for each module, or maybe to a dependency-tree-type diagram agnostic to each package manager?

Reply:
I like the idea of a visual diagram or table to help folks determine which packages they need.

Reply:
Since CocoaPods & SPM resolve the Braintree dependencies for a merchant (ex: if you include `BraintreeApplePay` it will handle including it's dependency `BraintreeCore`), we think a visual diagram could be more confusing to merchants.

For our Carthage users, we outline the required frameworks in the [dev docs](https://developers.braintreepayments.com/guides/drop-in/setup-and-integration/ios/v4#carthage). There's a few payment methods that need updated Carthage instructions, but we'll tackle them with our big v5 docs overhaul.